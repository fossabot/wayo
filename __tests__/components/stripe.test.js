import 'babel-polyfill';
import { shallowMount,createLocalVue,mount } from '@vue/test-utils';
import Stipe from '../../src/components/stripe';
import StipeButton from '../../src/components/stripe/_stripe-button.vue';

const LocalVue = createLocalVue();

describe('Snapshot', () => {
  test('stripe', () => {
    const Wrapper = mount(Stipe, {
      localVue: LocalVue
    });
    expect(Wrapper).toMatchSnapshot();
  });

  test('stripe button', () => {
    const Wrapper = mount(StipeButton, {
      localVue: LocalVue
    });
    expect(Wrapper).toMatchSnapshot();
  });
});