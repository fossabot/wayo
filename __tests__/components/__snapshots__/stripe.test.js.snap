// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot stripe 1`] = `<div class="wayo-stripe wayo-stripe_size_default"></div>`;

exports[`Snapshot stripe button 1`] = `<div class="wayo-stripe-button wayo-stripe-button_stroke"></div>`;
